src/main/java/net/sf/mzmine/util/StringCrypter.java

```diff
@@ -0,0 +1,123 @@
+package net.sf.mzmine.util;
```

Comment:
Could you please copy the license header to all new .java files?

Reply:
Done :-)